<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createCatalogWizard</class>
 <widget class="QWizard" name="createCatalogWizard">
  <property name="windowTitle">
   <string>Create Catalog Wizard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QtRc.qrc">
    <normaloff>:/image/images/catalog_new.png</normaloff>:/image/images/catalog_new.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <property name="options">
   <set>QWizard::DisabledBackButtonOnLastPage|QWizard::NoBackButtonOnStartPage</set>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="labelName">
      <property name="text">
       <string>N&amp;ame</string>
      </property>
      <property name="buddy">
       <cstring>lineName</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelTags">
      <property name="text">
       <string>Ta&amp;gs</string>
      </property>
      <property name="buddy">
       <cstring>lineTags</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineTags"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelDate">
      <property name="text">
       <string>&amp;Date</string>
      </property>
      <property name="buddy">
       <cstring>dateTimeEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QDateTimeEdit" name="dateTimeEdit"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="labelDesc">
      <property name="text">
       <string>D&amp;escription</string>
      </property>
      <property name="buddy">
       <cstring>lineDesc</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="2" rowspan="2">
     <widget class="QTextEdit" name="lineDesc"/>
    </item>
    <item row="4" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>58</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineDirPath">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="directoryCheck">
         <property name="text">
          <string>&amp;Fill catalog from directory</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="buttonChoseDir">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string notr="true">...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QLabel" name="labelReady">
      <property name="text">
       <string>If configuration is OK, so press the Next button and start cataloging.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelprogress">
      <property name="text">
       <string>Cataloging is started</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelStat">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Operation finished.

Press Finish button.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineName</tabstop>
  <tabstop>lineTags</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>lineDesc</tabstop>
  <tabstop>directoryCheck</tabstop>
  <tabstop>buttonChoseDir</tabstop>
  <tabstop>lineDirPath</tabstop>
 </tabstops>
 <resources>
  <include location="QtRc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>directoryCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineDirPath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>directoryCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonChoseDir</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
